cds-snc/covid-alert-server
# feat: add an endpoint to clean diagnosis keys

Add a new test-tools servlet
Add tests for test-tools servlet
Add test-tools servlet to retrieve service

Refactor some existing tests in the server namespace to use the new testhelpers logging functions
Fix an issue with how the event logger test was working

Write tests for the test_tools servlet
Write tests for the test_tools database code

## How to Test

Start a new monolith server ensure the env variable ENABLE_TEST_TOOLS is true 
Add some keys to the diagnosis_keys table 
Post to /clear-diagnosis-keys and pass a bearer token
Validate the diagnosis_keys table is empty 

This closes #331
